gpkit/model.py

```diff
@@ -300,8 +300,7 @@ def _solve(self, programType, solver, verbosity, skipfailures,
                            for (var, grid) in zip(sweep, sweep_grids)}
 
             if verbosity > 0:
-                print("Solving for %i variables over %i passes." %
-                      (len(self.variables), N_passes))
+                print("Solving over %i passes." % N_passes)
```

Comment:
But only if verbosity is 1, right? I think there's some utility in saying "x variables over y passes" if you're not planning on printing it out for each pass.

Maybe `if verbosity == 1:  print("Solving for %i variables" % len(self.variables))` ?